.github/workflows/test.yml

```diff
@@ -0,0 +1,17 @@
+on: [push, pull_request]
+name: Test
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: [1.14.x, 1.15.x]
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{ matrix.go-version }}
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Test
+      run: KAFKA_DISABLE=1 go test -v -race -timeout=1m ./...
```

Comment:
🌷 If it's not complicated to handle, I would use `richgo test -v -cover -timeout=1m ./... && echo OK || (echo FAIL && exit 1)` WDYT?

Reply:
Done in e8bbc91